content/boilerplates/notes/1.1.7.md

```diff
@@ -0,0 +1,14 @@
+## Security fixes
+
+- Fix issue where Mixer policy checks were incorrectly disabled for TCP, potentially allowing access to unauthorized resources ([Issue 13868](https://github.com/istio/istio/issues/13868)).  [TCP authorization is still in Alpha](https://istio.io/about/feature-stages/#security-and-policy-enforcement) and Alpha features may not be reliable or secure and are subject to backwards incompatible changes.
+
+## Bug fixes
+
+- Fix issue where two gateways with overlapping hosts, created at the same second, can cause Pilot to fail to generate routes correctly and lead to Envoy listeners stuck indefinitely at startup in a warming state.
+- Improve the robustness of the SDS node agent: if Envoy sends a SDS request with an empty `ResourceNames`, ignore it and wait for the next request instead of closing the connection ([Issue 13853](https://github.com/istio/istio/issues/13853)).
+- In prior releases Pilot automatically injected the experimental `envoy.filters.network.mysql_proxy` filter into the outbound filter chain if the service port name is `mysql`.  This was surprising and caused issues for some operators, so Pilot will now automatically inject the `envoy.filters.network.mysql_proxy` filter only if the `PILOT_ENABLE_MYSQL_FILTER` environment variable is set to `1` ([Issue 13998](https://github.com/istio/istio/issues/13998)).
+- Increase memory limit in the `ingressgateway` Helm chart to 1GB and add resource `request` and `limits` to the SDS node agent container to support HPA autoscaling.
```

Comment:
This should probably be in the enhancement section below.

Reply:
Moved it